fedimint-server/src/lib.rs

```diff
@@ -480,8 +480,11 @@ impl FedimintServer {
                 Ok(self.handle_step(step).await?)
             }
             (_, EpochMessage::RejoinRequest(epoch)) => {
-                info!("Requested to run {} epochs", epoch);
-                self.run_empty_epochs = min(NUM_EPOCHS_REJOIN_AHEAD, epoch);
+                self.run_empty_epochs += min(NUM_EPOCHS_REJOIN_AHEAD, epoch);
```

Comment:
This function, this field, this message - none of them have a comment. `NUM_EPOCHS_REJOIN_AHEAD` does have a comment "how many epochs ahead of consensus to rejoin".

I don't understand why we would rejoin 'ahead' of consensus? Like, to epochs that are not even made yet?

I don't understand. :(

Reply:
I'll try to add a better comment around that.